rules/brand-more-than-logo/rule.md

```diff
@@ -0,0 +1,74 @@
+<script async src="//www.instagram.com/embed.js"></script>
+**Video: Your brand is more than just a logo (40 sec)**
+
+## Key elements of brand identity
+
+### 1. Color Palette: The Emotional Trigger
+Your brand colors evoke specific emotions. Consistent use helps build recognition and trust. Examples include:
+- **Blue:** Trust and stability (e.g., IBM, Visa).
+- **Red:** Energy and excitement (e.g., Coca-Cola, Netflix).
+- **Green:** Growth and tranquility (e.g., Starbucks, Whole Foods).
+
+### 2. Typography: Communicating Personality
+Fonts do more than look good—they communicate your brand’s personality.
+- **Sans-serif fonts** (e.g., Helvetica): Modern and approachable.  
+- **Serif fonts** (e.g., Times New Roman): Traditional and reliable.  
+- **Custom fonts** (e.g., Coca-Cola’s script): Unique to the brand.
+
+### 3. Tone of Voice: Shaping Perception
+How you say things matters just as much as what you say. Examples of tone in action:
+- **Formal:** Financial institutions (e.g., Goldman Sachs).
+- **Conversational:** Consumer apps (e.g., Slack).
+- **Inspirational:** Athletic brands (e.g., Nike).
+
+## Brand guidelines ensure consistency
+
+Without guidelines, brand communication becomes inconsistent. A brand guideline includes:
+- **Color usage** (HEX, RGB values)
+- **Typography standards** (font sizes, spacing)
+- **Logo rules** (placement, acceptable variations)
+- **Imagery guidelines** (photo styles and iconography)
+- **Tone of voice examples** (proper messaging across platforms)
+
+::: info
+#### Case Study: Google's Seamless Brand Identity
```

Comment:
Hey @bettybondoc 

- [ ] 1. OPTIONAL - Could we find and add a link to an article that explains this one better?